src/braintree.ts

```diff
@@ -1,32 +1,64 @@
-import * as braintree from 'braintree-node'
+import {Gateway, GatewayOptions} from "../typings/braintree";
+const braintree = require('braintree')
```

Comment:
I've switched to using the official braintree library here as the promise based one was a bit pants